packages/amplify_authenticator/pubspec.yaml

```
@@ -1,7 +1,7 @@
 name: amplify_authenticator
 description: A prebuilt Sign In and Sign Up experience for the Amplify Auth category
-version: 0.1.0-rc.6
-homepage: https://github.com/aws-amplify/amplify-flutter/tree/release-candidate/packages/amplify_authenticator/
+version: 0.1.0
+homepage: https://ui.docs.amplify.aws/components/authenticator?platform=flutter
```

Let's add the issue tracker if we remove the GH homepage

```suggestion
homepage: https://ui.docs.amplify.aws/components/authenticator?platform=flutter
issue_tracker: https://github.com/aws-amplify/amplify-flutter/issues
```